README.md

```diff
@@ -703,6 +703,7 @@ Table of Contents
 
 ## Storage and Media Processing
 
+   * [cloudimage.com](https://cloudimage.com/) — Full image optimization and CDN service with 1500 POPs around the world. A variety of image resizing, compression, watermarking functions. Open source plugins for responsive images, 360 image making and image editing. Free monthly plan with 25GB of CDN traffic and 25GB of cache storage and unlimited transformations.
```

Comment:
Would it be better if we explain what POP means?

Reply:
Probably, updated it to 'Points of Presence'.